onnxmltools/convert/sklearn/shape_calculators/LinearClassifier.py

```diff
@@ -19,43 +19,33 @@ def calculate_sklearn_linear_classifier_output_shapes(operator):
     appear in this operator's output list, we should further generate a map storing all classes' probabilities.
 
     Allowed input/output patterns are
-        1. [N, C] ---> [N, 1], A sequence of map
+        1. [N, C] ---> [N], [N, E] (binary-class)
+        2. [N, 1], ..., [N, 1] ---> [N], [N, E] (binary-class)
+        3. [N, C] ---> [N], A sequence of map (multi-class)
+        4. [N, 1], ..., [N, 1] ---> [N], A sequence of map (multi-class)
 
     Note that the second case is not allowed as long as ZipMap only produces dictionary.
     '''
-    check_input_and_output_numbers(operator, input_count_range=1, output_count_range=[1, 2])
+    check_input_and_output_numbers(operator, input_count_range=[1, None], output_count_range=[1, 2])
```

Comment:
I don't think scikit-learn can support multiple inputs. Is it required by other packages?